dipy/fixes/argparse.py

```diff
@@ -1,4 +1,5 @@
-# emacs: -*- coding: utf-8; mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# emacs: -*- coding: utf-8; mode: python; py-indent-offset: 4;
+# indent-tabs-mode: nil -*-
```

Comment:
Sorry - I believe this needs to all be on one line for emacs to properly pick it up.